02 - Community Expansions/02.04 Auxiliary Tools by Category/Chrome Extensions.md

```diff
@@ -11,8 +11,8 @@ publish: true
 Official website: #placeholder/link
 Documentation: #placeholder/link
 Cost: #placeholder/tool
-Available for: #placeholder/tool %% Uncomment this and remove those that don't apply: [[Windows Tools|Windows]], [[MacOS Tools|MacOS]], [[Linux Tools|Linux]], [[Android Apps|Android]], [[iOS Apps|iOS]], [[iPadOS Apps]] %%
+Available for: [[Windows Tools|Windows]], [[MacOS Tools|MacOS]], [[Linux Tools|Linux]]
```

Comment:
I don't follow. I just removed the references to Android and iOS, since Chrome extensions in general are not available on mobile.

I haven't changed the location or anything else about the note?

Reply:
the note looks like a category file for chrome extensions to me?

Reply:
Yeah, I'm not sure when the change was introduced. But since you're adding changes to this note, we might as well apply the right template!

Reply:
Hmm, but thing is, there isn't any Auxiliary Tool Category Template yet to apply. 

Well, I can just as well create one, can I? 🙃

Reply:
all done